.github/workflows/test-workflow.yml

```diff
@@ -25,10 +25,32 @@ jobs:
         aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
         aws-region: us-west-2
 
+    - name: Deploy Amazon ECS task definition With Wait For Task Stopped Being False
+      uses: aws-actions/amazon-ecs-deploy-task-definition@1a69daea10712415b65b5c90f8c41b1b6b556ab5
+      with:
+        task-definition: task-definition-run-task.json
+        cluster: github-actions-deploy-task-def-integ-test
+        run-task: true
+        run-task-subnets: subnet-e5604fce, subnet-fe9355b4, subnet-c49431bc, subnet-392f9b64
+        run-task-security-groups: sg-067ebcde49c0f3ad8
+        run-task-launch-type: FARGATE
+        wait-for-task-stopped: false
+
+    - name: Deploy Amazon ECS task definition With Wait For Task Stopped Being True
+      uses: aws-actions/amazon-ecs-deploy-task-definition@1a69daea10712415b65b5c90f8c41b1b6b556ab5
+      with:
+        task-definition: task-definition-run-task.json
+        cluster: github-actions-deploy-task-def-integ-test
+        run-task: true
+        run-task-subnets: subnet-e5604fce, subnet-fe9355b4, subnet-c49431bc, subnet-392f9b64
+        run-task-security-groups: sg-067ebcde49c0f3ad8
+        run-task-launch-type: FARGATE
+        wait-for-task-stopped: true
+
     - name: Deploy Amazon ECS task definition
```

Comment:
nit: we can remove this to "Deploy Amazon ECS task definition to an ECS Service" for clarity.